[2025-02-06T11:59:44.175+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T11:59:44.209+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-01-01T06:34:30+00:00 [queued]>
[2025-02-06T11:59:44.251+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-01-01T06:34:30+00:00 [queued]>
[2025-02-06T11:59:44.252+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T11:59:44.314+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): first_task> on 2025-01-01 06:34:30+00:00
[2025-02-06T11:59:44.318+0000] {standard_task_runner.py:72} INFO - Started process 516 to run task
[2025-02-06T11:59:44.322+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-01-01T06:34:30+00:00', '--job-id', '2263', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpblftdudl']
[2025-02-06T11:59:44.323+0000] {standard_task_runner.py:105} INFO - Job 2263: Subtask first_task
[2025-02-06T11:59:44.454+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-01-01T06:34:30+00:00 [running]> on host f73fb7f0e18f
[2025-02-06T11:59:44.737+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T06:34:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T06:34:30+00:00'
[2025-02-06T11:59:44.740+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T11:59:44.740+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T11:59:44.741+0000] {logging_mixin.py:190} INFO - Current task name:first_task state:running start_date:2025-02-06 11:59:44.210465+00:00
[2025-02-06T11:59:44.742+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag and current dag run status:running
[2025-02-06T11:59:44.742+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T11:59:44.796+0000] {python.py:240} INFO - Done. Returned value was: 267
[2025-02-06T11:59:44.880+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T11:59:44.881+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-01-01T06:34:30+00:00, execution_date=20250101T063430, start_date=20250206T115944, end_date=20250206T115944
[2025-02-06T11:59:45.122+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-06T11:59:45.126+0000] {process_utils.py:132} INFO - Sending 15 to group 516. PIDs of all processes in the group: [516]
[2025-02-06T11:59:45.127+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 516
[2025-02-06T11:59:45.138+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-02-06T11:59:45.139+0000] {logging_mixin.py:190} WARNING - Exception ignored in: <function WeakKeyDictionary.__init__.<locals>.remove at 0x7f8a2a294860>
[2025-02-06T11:59:45.141+0000] {logging_mixin.py:190} WARNING - Traceback (most recent call last):
[2025-02-06T11:59:45.142+0000] {logging_mixin.py:190} WARNING -   File "/opt/conda/lib/python3.12/weakref.py", line 369, in remove
[2025-02-06T11:59:45.144+0000] {logging_mixin.py:190} WARNING -     def remove(k, selfref=ref(self)):
[2025-02-06T11:59:45.145+0000] {logging_mixin.py:190} WARNING -   File "/opt/conda/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3095, in signal_handler
[2025-02-06T11:59:45.148+0000] {logging_mixin.py:190} WARNING -     raise AirflowTaskTerminated("Task received SIGTERM signal")
[2025-02-06T11:59:45.150+0000] {logging_mixin.py:190} WARNING - airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2025-02-06T11:59:45.384+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T11:59:45.386+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T11:59:45.388+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag queued_at:2025-02-06 11:59:41.650413+00:00
[2025-02-06T11:59:45.389+0000] {logging_mixin.py:190} INFO - Task hostname:f73fb7f0e18f operator:PythonOperator
[2025-02-06T11:59:45.424+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=516, status='terminated', exitcode=0, started='11:59:43') (516) terminated with exit code 0
